opencensus-shim/src/test/java/io/opentelemetry/opencensusshim/OpenTelemetryPropagationComponentImplTest.java

```diff
@@ -33,8 +33,8 @@
 
   @Test
   public void implementationOfBinary() {
```

Comment:
Is this test still useful? Maybe we shouldn't test the impl anymore as we don't control it, that way if I need to fix opencensus for whatever reason, this doesn't silently break.

Reply:
I think we can deal with it if it happens - I wanted to have this so that if the assertion fails because the class is now `NoopBinaryFormatImpl` for whatever reason, we will know something significant needs to be done :P